server/lib/pencil_space_server/game.ex

```diff
@@ -0,0 +1,58 @@
+defmodule PencilSpaceServer.Game do
+  @moduledoc """
+  Coordinate the game overall: state, updates, process life-cycles etc.
+  """
+
+  alias PencilSpaceServer.Game.{Monitor, Supervisor}
+
+  def start do
+    name = Nanoid.generate()
+    case Supervisor.supervise({Monitor, name: ref(name)}) do
```

Comment:
MINOR: `Supervisor` here shadows _the Supervisor_ module. Maybe rename the alias to GameSupervisor or something else